tests/qml/tst_VPNTextArea.qml

```diff
@@ -0,0 +1,83 @@
+import QtQuick 2.3
```

Comment:
copyright/left mozilla header?

Reply:
yes - please add the standard mozilla header - copy from any other file

```
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 ```